packages/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -331,6 +371,33 @@ const useStyles = makeStyles({
       },
     };
   },
+  rootTransparent: theme => {
+    const buttonTokens = makeButtonTokens(theme);
+
+    return {
+      background: buttonTokens.transparent.background,
+      borderColor: buttonTokens.transparent.borderColor,
+      color: buttonTokens.transparent.color,
+
+      boxShadow: buttonTokens.transparent.shadow,
+
+      ':hover': {
+        background: buttonTokens.transparent.hovered?.background,
```

Comment:
This is a little confusing; why do buttonTokens have `transparent.hovered.background`, whereas alias tokens have `transparent.backgroundHover`?

Can you work with designers to create some consistency in patterns?

I think it would make more sense to have the alias color tokens match the button shape here:

```jsx
colors: {
   transparent: { // transparent color group
     background: '', // rested state color token
     hovered: { // transparent/hovered color group
       background: '', // hovered state color token
    }
  }
}
```

Reply:
I could try to, I don't think it's a hard conversation to have.